torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -24,6 +26,23 @@ def remote_device(module_rref):
         return param.device
 
 
+# RPC handler for querying __dict__ on the destination worker.
+def remote_module_attributes(remote_module):
+    return remote_module.__dict__
+
+
+# RPC handler for running forward on the destination worker.
+def remote_forward(remote_module, args):
+    return remote_module.forward(*args)
+
+
+# RPC handler for running forward_async on the destination worker.
+def remote_forward_async(remote_module, args):
+    # Since future cannot be pickled and sent over the RPC layer,
```

Comment:
Can we file an issue for that? Although I guess we can't really support this, besides maybe when the future is set with a value.

Reply:
Discussed with Pritam, the users are not supposed to send a future over RPC. Here the purpose is only testing whether the two generated methods are installed on the remote worker.